### PR TITLE
Update HelloWorld.java

### PR DESCRIPTION
**Pull request by vshashikumar**
Please merge the code changes into main branch

### CODE REVIEW

File: HelloWorld/src/HelloWorld.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 public class HelloWorld {
 
 	public static void main(String[] args) {
-		System.out.println("Hello World!");
+		System.out.println("Hello World");
```
Comment (bills-weberon) [inline HelloWorld/src/HelloWorld.java]: Why is it changed

File: HelloWorld/src/HelloWorld.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 public class HelloWorld {
 
 	public static void main(String[] args) {
-		System.out.println("Hello World!");
+		System.out.println("Hello World");
 
 	}
```
Comment (bills-weberon) [inline HelloWorld/src/HelloWorld.java:7]: Change the indentation

File: HelloWorld/src/HelloWorld.java (hunk 1 above)
Comment (vshashikumar) [reply HelloWorld/src/HelloWorld.java]: it is required

**Review by bills-weberon — APPROVED** [review]
Approved

**Pull request merged**